src/ts/main/src/main.ts

```diff
@@ -76,6 +77,34 @@ function main() {
         // for more about languagePluginLoader, see https://github.com/iodide-project/pyodide/blob/master/src/pyodide.js
         languagePluginLoader.then(() => {
 
+            //TODO: this needs to be a more complete python IO implementation,
+            // see https://github.com/python/typeshed/blob/5d553c9584eb86793cfa61019ddee5c7b62bc286/stdlib/2/typing.pyi#L320
+            (<any>window).fakeStdout = {
```

Comment:
nit: it's not fake, appStdout, consoleStdout, browserStdout...

Reply:
This code is not production quality - this really needs to be a validly-duck-typed python IO implementation.

Can I rename it in a separate branch?

Reply:
sure